### ISSUE TITLE
Default template is not processing markdown

### ISSUE THREAD

**Issue by coop2540** [MarcelKr/grav-theme-tail#9]
Hi, 
If I use the default template with this code:

```
{% extends 'partials/base.html.twig' %}

{% block content %}
<article class="max-w-xl mx-auto">
    <header>
        <h1>{{ page.header.title }}</h1>
    </header>

            {{ page.content }}
    </article>
{% endblock %}
```

Markdown is not processing. 
If I put {{page.content}} inside `<section class=prose>` like in the post template then the markdown works.

I'm doing something wrong ? or is a not defined css for default.html.twig and is for this that markdown is not processed?

Thanks!

**Comment by MarcelKr** [comment]
Hey Jordan, thanks for reporting the issue. Indeed, the default template was missing the `prose` class. I fixed it via e0f7944cbb7e5f8aa0a35c749a8226b7fbb476b5.

This fix is part of the 1.4.2 release which should soon be available via GPM.

**Comment by coop2540** [comment]
Thank you !!
